compiler/optimizer/OMROptimizer.cpp

```diff
@@ -1566,7 +1566,6 @@ int32_t OMR::Optimizer::performOptimization(const OptimizationStrategy *optimiza
             bool blocksArePending = false;
             while (subGroup->_num != endGroup && subGroup->_num != endOpts)
                {
-               OMR::Optimizations optNum = subGroup->_num;
```

Comment:
ok if it is dead that is fine - I was more looking at the meaning of the local itself - if unused that should be fine to delete.